package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
```

Comment:
- [ ] Kindly fix the error in the image below, I failed to test the app because it kept popping up every time I wanted to use the app.

![webpack](https://user-images.githubusercontent.com/53124548/223721203-98cab525-e6b8-4d6f-b1ed-79ebf16b2977.png)

Reply:
@elisha2kyakpo1 
Could you please tell me how to fix this?